build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -341,7 +341,9 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         [Target]
         public static BuildTargetResult BuildProjectsForNuGetPackages(BuildTargetContext c)
         {
-            if (CurrentPlatform.IsWindows)
+            string rid = c.BuildContext.Get<string>("TargetRID");
+
+            if (CurrentPlatform.IsWindows || rid == "ubuntu.14.04-arm" || rid == "ubuntu.16.04-arm" || rid == "tizen.4.0.0-armel" )
```

Comment:
It was only built on Windows before because we only ever published the NuGet packages from the Windows build, since that was the only build that signed the assemblies.

So it was a build optimization - nothing ever used/consumed these NuGet packages on non-Windows, so there was no reason to build them.